www/src/data/sidebars/doc-links.yaml

```diff
@@ -188,26 +188,26 @@
         - title: Styling Your Site
           link: /docs/styling/
           items:
-            - title: Using Layout Components
-              link: /docs/layout-components/
```

Comment:
We should add redirects for all of these moved/deleted pages in gatsby-node.js: https://www.gatsbyjs.org/contributing/docs-contributions/#docs-renaming-instructions

Reply:
Aaaah this is where we're dealing with these! I'll take care of it :slightly_smiling_face:

Reply:
Redirected Component CSS to the new "Component-scoped styles with CSS Modules" in 
ddc745b.